basta/bastamain.py

```diff
@@ -177,11 +177,11 @@ def BASTA(
 
                 # Always append the special weight for isochrones/tracks
                 if "isochrones" in gridtype.lower():
-                    grid_weights.append("massini")
```

Comment:
Actually, I was sort-of wrong. This is for isochones and the along-weight. However, it should still be called `"massini"` because the `apply_dweight` stuff expects that.